website/models.py

```diff
@@ -46,7 +46,14 @@ def generate_uuid_for_company(apps, schema_editor):
         obj.save()
 
 
-class Company(models.Model):
+class UUIDModel(models.Model):
+    company_id = models.UUIDField(default=uuid.uuid4, editable=False, unique=True)
```

Comment:
Almost there -- as I mentioned you would need to have a unified name for the field. This way we would be able to use it in any model. If you name it `company_id` it fits companies context only

```suggestion
    uuid = models.UUIDField(default=uuid.uuid4, editable=False, unique=True)
```

Reply:
It's also a good idea to extract `UUIDModel` to a separate module accessible from any of our Django apps.

Reply:
@arkid15r sir ,  we need company_id field in company model for solve the error , now if we name it "uuid" the error not gonna fix . 
(basically the logic is coded by previous developer ,for now we have to consider this field not as a common field cause backend part is already done by using company_id so now maybe we don't need 
     `class UUIDModel(models.Model):
    company_id = models.UUIDField(default=uuid.uuid4, editable=False, unique=True)

    class Meta:
        abstract = True`
 
 
 so for the company,  we can just add :
    `company_id = models.UUIDField(default=uuid.uuid4, editable=False, unique=True)`
    
    
    now waiting for confirmation from you for make it like before

Reply:
> @arkid15r sir , we need company_id field in company model for solve the error , now if we name it "uuid" the error not gonna fix . (basically the logic is coded by previous developer ,for now we have to consider this field not as a common field cause backend part is already done by using company_id so now maybe we don't need

Let's take a look at the logic you've just mentioned. Is there an easy way to change it to lookup the entity based on `uuid` fiedl instead of `company_id`?

Reply:
> Let's take a look at the logic you've just mentioned. Is there an easy way to change it to lookup the entity based on `uuid` fiedl instead of `company_id`?

@arkid15r  sir ,those logic are huge , I have already checked and if I start changing on those stuff that will take huge time

Reply:
Apparently, I'm not gonna make you refactor a bunch of code just to have a unified field name. However, I'd like to understand what the logic you're referring to is. As I couldn't figure out any details neither from the PR nor from the [related issue](https://github.com/OWASP-BLT/BLT/issues/2380). Is it about `CompanyViewSet`?

Reply:
@arkid15r ok sir now I update the issue and explain everything actually what I'm gonna fix

Reply:
@arkid15r , can you please check the description from : #2380

Reply:
Well, something is wrong there for sure. The `company` should not have a field `company_id`  in the first place (it's already in the company context so simple `id` would be enough -- and actually `id` field is already there).

It seems that we need to fix the `Company` form submission logic instead of introducing the new field.
Unfortunately I don't have time to troubleshoot the #2380 today. I'm going to look into this later.

Reply:
@arkid15r , ok then sir I will try to fix the error my own , then will ask suggestion from you